services/cadip/config/cadip_ws_config.yaml

```diff
@@ -76,3 +76,118 @@ cadip:
     metadata_mapping:
```

Comment:
Idem, do we need to update this file to rs-demo in https://github.com/RS-PYTHON/rs-demo/tree/main/local-mode/main/config ?